src/xcode/ENA/HealthCertificateToolkit/Tests/HealthCertificateToolkitTests/ProofCertificateDownloadTests.swift

```diff
@@ -0,0 +1,327 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+
+import XCTest
+import SwiftCBOR
+@testable import HealthCertificateToolkit
+
+final class ProofCertificateDownloadTests: XCTestCase {
+
+    func test_When_fetchProofCertificate_Then_CorrectDataIsReturned() throws {
+        let proofCertificateDownload = ProofCertificateDownload()
+        let certificateAccess = DigitalGreenCertificateAccess()
+
+        let httpServiceStub = HTTPServiceStub(completions: [
+            HTTPServiceStub.Completion(
+                data: cborTestData,
+                response: makeResponse(withStatusCode: 200),
+                error: nil
+            )
+        ])
+
+        let resultExpectation = expectation(description: "Fetch should return a result.")
+
+        proofCertificateDownload.fetchProofCertificate(for: [testData.input], with: httpServiceStub) { [weak self] result in
+            guard case let .success(_proofCertificateData) = result,
+                  let proofCertificateData = _proofCertificateData else {
+                XCTFail("Success expected.")
+                return
+            }
+
+            let result = certificateAccess.extractDigitalGreenCertificate(from: proofCertificateData)
+            guard case let .success(proofCertificate) = result else {
+                XCTFail("Success expected.")
+                return
+            }
+
+            XCTAssertEqual(proofCertificate, self?.testData.output)
+            resultExpectation.fulfill()
+        }
+
+        waitForExpectations(timeout: 3.0)
```

Comment:
```suggestion
        waitForExpectations(timeout: .short)
```

Reply:
.short is not available in the package. Also a point for discussion.

Reply:
:( okay